.travis.yml

```diff
@@ -1,15 +1,13 @@
 language: go
-sudo: false
 go:
-- 1.4.2
 - 1.5
 install:
-- export GOPATH=${TRAVIS_BUILD_DIR}/Godeps/_workspace:$GOPATH
```

Comment:
Because of vendoring capabilities in golang 1.5, that basically removes the need for any GOPATH monkeying, or relying on calls to godep itself during normal build/test operations.
Furthermore, leveraging the make infrastructure let us keep that complexity out of the specific CI systems definition files.